<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoEstoque</class>
 <widget class="QDialog" name="fm_gestaoEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestão de Estoque</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/imagens/img/images.png</normaloff>:/imagens/img/images.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_gestaoprodutos">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>631</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Codigo do produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_codproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Descrição do produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_descproduto"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Quantidade em estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_qtdestoque"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Valor de compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valorcompra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Valor de venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_valovenda"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_fornecedor"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_novoproduto">
          <property name="text">
           <string>novo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gravanovoproduto">
          <property name="text">
           <string>gravar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_novoproduto">
    <attribute name="title">
     <string>Gestão dos Produtos</string>
    </attribute>
    <widget class="QTableWidget" name="tw_ge_produtos">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>381</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>120</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar por:</string>
     </property>
     <widget class="QRadioButton" name="rb_ge_codigo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_ge_produto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Produto</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>641</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_ge_gravarF">
        <property name="text">
         <string>Gravar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_excluir">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>284</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Codigo do Produto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_codproduto"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Produto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_produto"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Quantidade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_qtd"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Valor Compra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_valcompra"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Valor Venda</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_valvenda"/>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Fornecedor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_ge_fornecedor"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Filtar</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="txt_ge_filtrar"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ge_filtrar">
          <property name="text">
           <string>ok</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
